gateway_code/control_nodes/cn_iotlab/__init__.py

```diff
@@ -34,11 +35,10 @@
 LOGGER = logging.getLogger('gateway_code')
 
 
-class ControlNodeIotlab(object):
+class ControlNodeIotlab(ControlNode):
     """ Control Node implemenation """
```

Comment:
s/implemenation/implementation/ (typo not related to your changes)

Reply:
Yes, I believe it's fixed somewhere else,I removed unrelated clutter from this PR